tests/phpunit/api/v4/Utils/CoreUtilTest.php

```diff
@@ -45,6 +46,11 @@ public function testGetApiNameFromTableName() {
       ->addValue('title', uniqid())
       ->addValue('extends', 'Contact')
       ->addValue('is_multiple', TRUE)
+      ->addChain('fields', CustomField::save()
+        ->addDefault('html_type', 'Text')
+        ->addDefault('custom_group_id', '$id')
+        ->addRecord(['label' => 'MyField1'])
+      )
```

Comment:
General patch makes sense.

What's the connection between `Entity::getInfo()['table_name']` and the ability to join `CustomGroup::create()` to `CustomField::save()`? (I would have expected that the only magic in the chain is based on the FK between CG/CF.)

Reply:
@totten it turns out APIv4 only bothers to create a virtual entity from a custom field set if it contains fields. Which makes sense. So this test had been testing a scenario that wouldn't fully work.